stories/tree/tree.stories.js

```diff
@@ -1,897 +1,1019 @@
             </div>
         </div>
-        <ul class="fd-tree__group fd-tree__group--sublevel-1 is-hidden" role="group" aria-hidden="true">
-            <li class="fd-tree__item" role="treeitem" id="Bxd8s850" aria-expanded="false">
-                <div class="fd-tree__row">
-                    <div class="fd-tree__col fd-tree__col--control">
-                        <button class="fd-tree__control" aria-label="Expand" aria-controls="Bxd8s850"
-                        aria-pressed="false" onclick="onControlClick('Bxd8s850');"></button>
-                        Second Level
-                    </div>
-                    <div class="fd-tree__col">
-                        Secondary Value
-                    </div>
-                    <div class="fd-tree__col">
-                        Secondary Value
-                    </div>
-                    <div class="fd-tree__col">
-                            Inactive
-                    </div>
-                    <div class="fd-tree__col fd-tree__col--actions">
-                        <div class="fd-popover fd-popover--right">
-                            <div class="fd-popover__control">
-                                <button class="fd-button fd-button--transparent sap-icon--overflow" aria-controls="lklkj3"
-                                aria-haspopup="true" aria-expanded="false" aria-label="More" onclick="onControlClick('lklkj3');"></button>
+    </li>
+    <li role="treeitem" aria-level="1" class="fd-tree__item">
+        <div class="fd-tree__item-container">
+            <div class="fd-tree__content fd-tree__content--wrap">
+                <span class="fd-tree__text">Level 1 with very long text and wrapping behaviour ... perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem.</span>
+            </div>
+        </div>
+    </li>
+</ul>
+`;
+
+treeWithExpandedLevel1.parameters = {
+    docs: {
+        iframeHeight: 950,
+        storyDescription: `
+When Level 1 is expanded to show Level 2, Level 2 is indented by 1.5rem.
+Only the root tree (ul element) needs to have the <code class="docs-code">.expanded-level-*</code> modifier class in order to keep track of which level is currently expanded.
```

Comment:
I think instead of "in order to keep track," you can say **"Only the root tree (ul element) must have the <code class="docs-code">.expanded-level-*</code> modifier class to indicate which level is currently expanded."**